app/routes/explore.js

```diff
@@ -79,6 +79,20 @@ export default class ExploreRoute extends Route {
         val  : params.is_online
       });
     }
+    if (!params.has_image) {
+      filterOptions.push({
+        name : 'original-image-url',
+        op   : 'ne',
+        val  : null
+      });
```

Comment:
the condition used inside is for checking if it has a image. And it is named like "not has_image".

Reply:
When has_image=null, this condition become true and only events with images are shown. 
When has_image=false, this condition hold false and all events whether image is present or not will be shown.

Reply:
That's opposite logic though. has_image=true should only show events with images

Reply:
> That's opposite logic though. has_image=true should only show events with images

Now i am little bit confused. 
Please correct me if i am wrong. 
Desired result:----> when has_image=null, all events shown whether it have image or not. 
When has_image=true, events with images must be shown.

Reply:
Yes